core/src/main/groovy/noe/common/utils/Cmd.groovy

```diff
@@ -821,12 +821,10 @@ public class Cmd {
      */
     Process psProc = psCommand.execute()
     Process proc2 = ['grep', identifier].execute()
-    // This is a big assumption, but I guess both EAP and Catalina have .sh start scripts...
-    Process proc3 = ['grep', '.sh'].execute()
-    Process proc4 = ['grep', '-v', 'grep'].execute()
```

Comment:
@sgala I see that proc4 was removed and proc3 become proc4. What's the idea behind this?

Reply:
In our use cases, piping through `grep .sh` destroys the result, as the startup script is executed using `eval exec', first creating a new shell and then substituting it by the `java ... Bootstrap` call.

I'd guess that this is also true in your use cases, but I couldn't find information anywhere.

If we could have a list of use cases we could refactor this code so that it actually works. Now it fails in a lot of cases where it should work, like most if not all our use cases.

`ps -ef | grep 1471948789` produces one line with our java server process, `ps -ef | grep 1471948789| grep .sh` produces an empty result.

To answer completely Matus question, if I'm removing the "proc3" in a pipeline, it made sense to rename proc4 as proc3, this is the reason why.